spec/fixtures/ai-proxy/unit/real-stream-frames/openai/llm-v1-completions.txt

```diff
@@ -0,0 +1 @@
+data: {"choices": [{"finish_reason": null,"index": 0,"logprobs": null,"text": "the answer"}],"created": 1711938803,"id": "cmpl-991m7YSJWEnzrBqk41In8Xer9RIEB","model": "gpt-3.5-turbo-instruct","object": "text_completion"}
```

Comment:
As mentioned in `spec/03-plugins/38-ai-proxy/01-unit_spec.lua` file, if we don't do test for openai, do we still need to add this file?

Reply:
This both a mistake, I'll fix them both.